src/test/py/bazel/bazel_windows_cpp_test.py

```diff
@@ -854,7 +854,24 @@ def testCcCompileWithTreeArtifactAsSource(self):
     ])
     exit_code, _, stderr = self.RunBazel(['build', '//:main'])
     self.AssertExitCode(exit_code, 0, stderr)
-
+    
+  def testBuild32BitCppBinaryWithMsvcCL(self):
+    self.CreateWorkspaceWithDefaultRepos('WORKSPACE')
+    self.ScratchFile('BUILD', [
+        'cc_binary(',
+        '  name = "main",',
+        '  srcs = ["main.cc"],',
+        ')',
+    ])
+    self.ScratchFile('main.cc', [
+        'int main() {',
+        '  return 0;',
+        '}',
+    ])
+    exit_code, _, stderr = self.RunBazel([
+        'build', '-s', '--cpu=x64_x86_windows', '//:main'
+    ])
+    self.AssertExitCode(exit_code, 0, stderr)
```

Comment:
`-s` will print the command Bazel executed, you can get the content in stderr. Can you check here we are actually using the compiler for 32 bit?  Something like line 773.